elk/extensions/recursive_ccs/train.py

```diff
@@ -0,0 +1,110 @@
+import csv
```

Comment:
It's not clear why this entire file is necessary. It seems to be mostly copied over from the primary `train.py`. Could we add a flag or a subparser to the main command instead?